JSDemos/menuMeta.json

```diff
@@ -3931,6 +3931,13 @@
               "/Models/Product.cs",
               "/Models/SampleData/Products.cs"
             ]
+          },
+          {
+            "Title": "Multiline",
+            "Name": "Multiline",
+            "Widget": "Toolbar",
+            "Badge": "New",
+            "DemoType": "Web"
```

Comment:
What is this prop necessary for?

Reply:
DemoType prop is responsible for the demo container type and adaptivity switcher. You can see it [here](https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/Overview/React/Light/) and [here](https://js.devexpress.com/Demos/WidgetsGallery/Demo/TreeList/Adaptability/React/Light/).